modules/trusted-firmware-m/CMakeLists.txt

```diff
@@ -24,6 +24,9 @@ set(TFM_VALID_PARTITIONS
 # BINARY_DIR: The location where the build outputs will be written
 # BOARD: The string identifying the board target for TF-M (AN521, etc.)
 # CMAKE_BUILD_TYPE: The TF-M build type to use, (Debug, Release, etc.)
+# PSA_TEST_SUITE: A PSA test suite to add, choose one of
```

Comment:
@oyvindronningstad can you only add a single PSA test suite?

Reply:
Yes, that's a limitation in psa-arch-tests